app/cdap/components/NamespaceAdmin/store/ActionCreator.ts

```diff
@@ -106,6 +111,64 @@ export function getPreferences(namespace) {
     });
 }
 
+export function getDrivers(namespace) {
+  const requestNamespace = namespace ? namespace : getCurrentNamespace();
+
+  MyCDAPVersionApi.get()
```

Comment:
This should be available in the VesionStore? I have made the change in main.js to load rest of the children after the CDAPVersion has been updated in the VersionStore.

Reply:
was that a recent change?

Reply:
Sorry I missed that it is available in the feature branch for connection management. https://github.com/cdapio/cdap-ui/pull/16/files#diff-270f5027c1d7025aa0c32f3dd64a385c6143083416f3b3c3bb230874a663416aR206-R214. 

Just noticed that this change is not based off of the feature branch. We can fetch the version from API here for now. Should be fine.